blocks/api/registration.js

```diff
@@ -202,12 +215,26 @@ export function setDefaultBlockName( name ) {
 /**
  * Retrieves the default block name.
  *
- * @return {?string} Blog name.
+ * @return {?string} Block name.
  */
 export function getDefaultBlockName() {
 	return defaultBlockName;
 }
 
+/**
+ * Retrieves the expected default block for the post format.
+ *
+ * @param	{string} postFormat Post format
+ * @return {string}            Block name.
+ */
+export function getDefaultBlockForPostFormat( postFormat ) {
```

Comment:
This function doesn't return a block, it returns a block name. It should have been called `getDefaultBlockNameForPostFormat`. Also see preceding function `getDefaultBlockName`.